src/xrUICore/TrackBar/UITrackBar.h

```diff
@@ -34,6 +34,9 @@ class XRUICORE_API CUITrackBar : public CUI_IB_FrameLineWnd, public CUIOptionsIt
     void SetOptIBounds(int imin, int imax);
     void SetOptFBounds(float fmin, float fmax);
 
+    CUIStatic* m_static;
```

Comment:
xr_unique_ptr?

Reply:
Never worked with unique_ptr.

Reply:
No need for this. S.T.A.L.K.E.R. UI is completely filled with raw pointers.

Reply:
And that's sad.